addons/escoria-ui-simplemouse/game.gd

```diff
@@ -87,9 +87,9 @@ func left_click_on_inventory_item(inventory_item_global_id: String, event: Input
 		var item = escoria.object_manager.get_object(
 			inventory_item_global_id
 		).node
-		if item.get_node("sprite").texture:
+		if item.get_sprite().texture:
```

Comment:
Maybe add a safety check in here like if item.has_method("get_sprite") or is it granted that we have this method at all times?

Reply:
Done